templates/zerver/confirm_continue_registration.html

```diff
@@ -0,0 +1,46 @@
+{% extends "zerver/portico_signup.html" %}
+
+{% block portico_content %}
+<div class="app register-page">
+    <div class="app-main register-page-container">
+        <div class="center-container">
+            <div class="center-block">
+                <div class="register-form">
+                    <p class="lead">
+                        <div class="register-page-header">{{ _("Account not found!") }}…</div>
+                    </p>
+                    {% if invalid_email %}
+                    <p>
+                        {% trans %}
+                        Please click the following button if you wish to register.
+                        {% endtrans %}
+                    </p>
+                    <a href='/register/' class="button-new sea-green">Register</a>
+                    {% else %}
+                    <p>{% trans %}You are trying to login with {{ email }}. If this is correct,
+                    you can register with this email address.{% endtrans %}</p>
```

Comment:
Maybe "Your attempted to login using {{ email }}, but {{ email }} does not have a Zulip account on this server.  If you'd like, you can try to register a new account with this email address."